app/pb_kits/playbook/pb_currency/_currency.scss

```diff
@@ -0,0 +1,50 @@
+
+  [class^=pb_currency_kit]{
+    &[class*=_center]  {
+      & > [class^=pb_caption],
+      & > [class^=pb_title],
+      & > [class^=pb_body] {
+        text-align: center;
+      }
+
+    }
+    &[class*=_right]  {
+      & > [class^=pb_caption],
+      & > [class^=pb_title],
+      & > [class^=pb_body] {
+        text-align: right;
+      }
+    }
+
+  }
+
+  [class^=pb_currency_wrapper] {
+    display: flex;
+
+    & > [class*=dollar_sign] {
+      display: flex;
+      align-items: flex-start;
+    }
+    & > [class*=unit] {
+      display: flex;
+      align-items: flex-end;
+    }
+    & > [class*=pb_currency_value] {
+      line-height: .9;
+      }
+
+  }
+
+  [class^=pb_currency_kit_center] {
```

Comment:
The playbook kit class is generated from the props, so the class can easily change depending on which props are passed.  This is why we do the *if matches* type css syntax (Example `[class^=pb_currency_kit]` and `[class*=pb_currency_value]`).

To make it less rigid, we should not search for `pb_currency_kit_center`, but instead search for a class that contains `_center` within the element `.pb_currency_kit`.

Like: 
```scss
[class^=pb_currency_kit] {
  &[class*=_center]  {
    ...
  }
}
```

Full Example:
```scss
[class^=pb_currency_kit]{
  &[class*=_center]  {
    & > [class^=pb_caption],
    & > [class^=pb_title],
    & > [class^=pb_body] {
      text-align: center;
    }
    
    // Move alignment scss here
    & > [class*=pb_currency_wrapper] {
      justify-content: center;
    }
  }
  
  &[class*=_right]  {
    & > [class^=pb_caption],
    & > [class^=pb_title],
    & > [class^=pb_body] {
      text-align: right;
    }

    // Move alignment scss here
    & > [class*=pb_currency_wrapper] {
      justify-content: flex-end;
    }
  }

}

[class^=pb_currency_wrapper] {
  display: flex;

  & > [class*=dollar_sign] {
    display: flex;
    align-items: flex-start;
  }
  & > [class*=unit] {
    display: flex;
    align-items: flex-end;
  }
  & > [class*=pb_currency_value] {
    line-height: .9;
  }
}
```

Reply:
makes senses.. Done!